.gitignore

```diff
@@ -63,4 +61,4 @@ annotation_data/model/model_999.pth
 
 .circleci/habitat_docker/conda.txt
 .circleci/habitat_docker/requirements.txt
-.circleci/habitat_docker/Replica-Dataset
\ No newline at end of file
+.circleci/habitat_docker/Replica-Dataset
```

Comment:
we can also nuke this for now? It is TBD where Replica or other scenes will live right now.